app/routes/endemics/timings.js

```diff
@@ -0,0 +1,42 @@
+const Joi = require('joi')
+const urlPrefix = require('../../config/index').urlPrefix
+const {
+  endemicsNumbers,
+  endemicsTimings,
+  endemicsOfferRejected,
+  endemicsDeclaration
+} = require('../../config/routes')
+
+const backLink = `${urlPrefix}/${endemicsNumbers}`
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${urlPrefix}/${endemicsTimings}`,
+    options: {
+      handler: async (request, h) => {
+        return h.view('endemics/timings', {
+          backLink
+        })
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: `${urlPrefix}/${endemicsTimings}`,
+    options: {
+      validate: {
```

Comment:
Same as line number 1, I believe this screen doesn't need validation because there are only two buttons without any kind of input element.